moveit_core/collision_detection/src/allvalid/collision_env_allvalid.cpp

```diff
@@ -36,10 +36,11 @@
 
 #include <moveit/collision_detection/allvalid/collision_env_allvalid.h>
 #include <moveit/collision_detection/allvalid/collision_detector_allocator_allvalid.h>
-#include "rclcpp/rclcpp.hpp"
+#include <rclcpp/logging.hpp>
 
 // Logger
-static const rclcpp::Logger LOGGER = rclcpp::get_logger("moveit_collision_detection.collision_world_allvalid");
+static const rclcpp::Logger LOGGER =
+    rclcpp::get_logger("moveit_core.moveit_collision_detection.collision_env_allvalid");
```

Comment:
```suggestion
    rclcpp::get_logger("moveit.core.collision_detection.allvalid");
```

Reply:
Technically, this doesn't follow the discussed pattern directly. But since the logger name is unique to the file, I would agree with this or `moveit.core.collision_detection.env_allvalid`.